components/Hooks/useEvents.ts

```diff
@@ -0,0 +1,42 @@
+import { useQuery } from "@tanstack/react-query";
+
+interface Event {
+  id: number;
+  title: string;
+  descripstion: string;
+  cover: string;
+  coverPlaceholder: string;
+  eventType: string;
+  eventStatusType: string;
+  location: string;
+  startTime: string;
+  endTime: string;
+  thumbnail: string;
+  totalCapacity: number;
+  company: Object;
+  registrationCount: number;
+  activationTime: string;
+  isAdmitted: boolean;
+  survey: string;
+  requireAuth: boolean;
+}
```

Comment:
Could also be moved to a separate types.ts file as mentioned above

Reply:
I fixed.